pkgs/development/compilers/orc/default.nix

```
@@ -12,6 +12,11 @@ in stdenv.mkDerivation rec {
     sha256 = "1w0qmyj3v9sb2g7ff39pp38b9850y9hyy0bag26ifrby5f7ksvm6";
   };
 
+  patchPhase = lib.optionalString stdenv.isAarch32 ''
```

```suggestion
  postPatch = lib.optionalString stdenv.isAarch32 ''
```
better to add a command, than override the phase in its entirety

---

Done, moved to postPatch.